pkg/virt-operator/resource/apply/core.go

```diff
@@ -561,11 +691,13 @@ func (r *Reconciler) createOrUpdateKubeVirtCAConfigMap(queue workqueue.TypedRate
 	version, imageRegistry, id := getTargetVersionRegistryID(r.kv)
 	injectOperatorMetadata(r.kv, &configMap.ObjectMeta, version, imageRegistry, id, true)
 
+	data := ""
 	obj, exists, _ := r.stores.ConfigMapCache.Get(configMap)
-
 	if !exists {
-		configMap.Data = map[string]string{components.CABundleKey: string(cert.EncodeCertPEM(caCert.Leaf))}
-
+		for _, externalCert := range externalCACerts {
+			data = data + string(cert.EncodeCertPEM(externalCert.Leaf))
+		}
+		configMap.Data = map[string]string{components.CABundleKey: data}
```

Comment:
Don't you need to add `caCert`? Or maybe set `data` to it above instead of empty string if order doesn't matter.
```suggestion
		data = data + string(cert.EncodeCertPEM(caCert.Leaf))
		configMap.Data = map[string]string{components.CABundleKey: data}
```

Reply:
Good point. not sure why I didn't catch that. Let me fix it.